### PR TITLE
Add corne keyboard

### PR DESCRIPTION
**Pull request by foostan**
I designed a new keyboard.
Corne keyboard(crkbd), a split keyboard with 3x6 vertically staggered keys and 3 thumb keys.

### CODE REVIEW

File: keyboards/crkbd/keymaps/default/keymap.c [hunk 1]
```diff
@@ -0,0 +1,246 @@
...
+#define KC_LSAD  RGB_SAD
+#define KC_LVAI  RGB_VAI
+#define KC_LVAD  RGB_VAD
+#define KC_LSMOD RGB_SMOD
+#define KC_CTLTB CTL_T(KC_TAB)
+#define KC_GUIEI GUI_T(KC_LANG2)
+#define KC_ALTKN ALT_T(KC_LANG1)
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+  [_QWERTY] = KEYMAP_kc( \
```
Comment (mechmerlin) [inline keyboards/crkbd/keymaps/default/keymap.c]: please change all instances of the `KEYMAP_kc` macro to `LAYOUT_kc`

File: keyboards/crkbd/rev1/rev1.h [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+#ifdef USE_I2C
+#include <stddef.h>
+#ifdef __AVR__
+	#include <avr/io.h>
+	#include <avr/interrupt.h>
+#endif
+#endif
+
+//void promicro_bootloader_jmp(bool program);
+#define KEYMAP( \
```
Comment (mechmerlin) [inline keyboards/crkbd/rev1/rev1.h]: please change all instances of the `KEYMAP` macro to `LAYOUT`

**Comment by mechmerlin** [comment]
Can you explain what your keylogger file is supposed to do? =P

File: keyboards/crkbd/serial.c [hunk 3]
```diff
@@ -0,0 +1,238 @@
...
+#define F_CPU 16000000
+#endif
+
+#include <avr/io.h>
+#include <avr/interrupt.h>
+#include <util/delay.h>
+#include <stdbool.h>
+#include "serial.h"
+
+#ifdef USE_SERIAL
```
Comment (drashna) [inline keyboards/crkbd/serial.c]: This should be `#ifndef USE_I2C`

**Comment by foostan** [comment]
> Can you explain what your keylogger file is supposed to do? =P

keylogger is recorder of key input, and output to OLED.

img_3818

File: keyboards/crkbd/serial.c (hunk 3 above)
Comment (foostan) [reply keyboards/crkbd/serial.c]: This file includes logic code for serial only.
Why cheking the flag of I2C here?

**Comment by foostan** [comment]
@mechmerlin @drashna Thank you reviews! Fixed some code and commented.

**Comment by drashna** [comment]
Thanks!

**Pull request merged**